.github/workflows/main.yml

```diff
@@ -204,30 +204,9 @@ jobs:
       run: bundle exec rake serialized_size:topgems
 
   memcheck:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
     steps:
     - uses: actions/checkout@v4
-    - name: Install libc6-dbg
-      run: |
-        sudo apt-get update
-        sudo apt-get install -y libc6-dbg
```

Comment:
I'm not sure if we still need this and/or if we need to install valgrind from apt, letting the CI find out

Reply:
valgrind needs to be installed, it's not in the image and it depends on `libc6-dbg`: https://github.com/ruby/prism/actions/runs/11391934879/job/31696848683?pr=3185#step:3:35